### ISSUE TITLE
Not compatible with sphinx 1.7

### ISSUE THREAD

**Issue by ceball** [pyviz/nbsite#35]
Which came out a few days ago.

<img width="1057" alt="screen shot 2018-02-15 at 12 08 51 pm" src="https://user-images.githubusercontent.com/1929/36255999-02ae0aaa-1249-11e8-82ef-466a6d6612b3.png">

**Comment by jlstevens** [comment]
Just to say I noticed this and had to use ``sphinx=1.6`` in the conda command. I'll also mention that I needed to updated nbconvert (is there an issue for that?)

**Comment by ceball** [comment]
sphinx is less than 1.7 in https://github.com/pyviz/nbsite/blob/master/setup.py#L27 but nbconvert does not have a min version specified. PR welcome :)

The conda package on pyviz/label/dev includes this specification, but the pip package doesn't (only releases go onto pypi).

Note that nbsite's build is currently broken since I moved the repo and started to reorganize its building...until I realized I needed a new project for that ;) Will be fixing it up soon.
